packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -69,11 +70,11 @@ function handleUpdate(e: boolean): void {
     </div>
 
     {#if category === 'developers'}
-      <DevelopersFeedback onCloseForm={hideModal} contentChange={handleUpdate}/>
+      <DevelopersFeedback close={hideModal} contentChange={handleUpdate}/>
     {:else if category === 'bug'}
-      <GitHubIssueFeedback onCloseForm={hideModal} category="bug" contentChange={handleUpdate}/>
+      <GitHubIssueFeedback close={hideModal} category="bug" contentChange={handleUpdate}/>
```

Comment:
question: why onCloseForm is renamed to close

note: close is meaningless, onclose is meaningful